gen3_util/files/cli.py

```diff
@@ -24,20 +25,38 @@ def files_ls(config: Config):
 
 
 @file_group.command(name="cp")
-@click.argument('from')
-@click.argument('to')
+@click.argument('from_', )
+@click.argument('to_')
+@click.option('--worker_count', default=10, show_default=True,
+              help="Number of worker processes")
+@click.option('--ignore_state', default=False, is_flag=True, show_default=True,
+              help="Upload file, even if already uploaded")
+@click.option('--project_id', default=None, required=True, show_default=True,
+              help="Gen3 program-project")
+@click.option('--source_path', required=False, default=None, show_default=True,
+              help='Path on local file system')
+@click.option('--disable_progress_bar', default=False, is_flag=True, show_default=True,
+              help="Show progress bar")
```

Comment:
Might "Show progress bar" be "Hide progress bar" if it's describing the `--disable_progress_bar` flag?